Source/pyboy/window/base_window.py

```diff
@@ -57,11 +58,17 @@ def render_screen(self, lc):
     def blank_screen(self):
         pass
 
-    def getscreenbuffer(self):
+    def get_screen_buffer(self):
+        raise NotImplementedError()
+
+    def get_screen_buffer_as_nparray(self):
```

Comment:
`nparray` is a bit confusing, as the array type is `np.ndarray`. `ndarray` may be more clear and consistent (and matches what I seem to remember in similar situations).

Reply:
I think you are right. It was called that at one point. Don't remember why I switched.